src/modules/endpoint/ResourceURLFinder.js

```diff
@@ -55,8 +56,8 @@ export default class ResourceURLFinder {
       return {
         ...acc,
         ...keyBy(
-          Array.from(urlSet).map(url => [url, path]),
-          ([url, path]) => buildFilePath(url, path)
+          Array.from(urlSet).map(url => [url, path, hashUrl(url)]),
```

Comment:
Why did you move the hashing to the ResourceURLFinder?

Reply:
The hash URL was missing in the return type of fetchResources(). In order to fix this I had to generate it earlier.